tests/bug-5220/bug-5220-5/test.rules

```diff
@@ -0,0 +1 @@
+alert http $HOME_NET any -> $EXTERNAL_NET any (flow:established,to_server; http.method; content:"POST"; fast_pattern; http.request_body; base64_decode:bytes 28; base64_data; bsize:=21; content:"something"; classtype:bad-unknown; sid:123; rev:1;)
```

Comment:
There's something diff b/w `6.0.x` and later versions that tests 5, 6 and 7 fail on `6.0.x`

Reply:
Fails on `6.0.x` because `bsize:=..` doesn't exist in `6.0.x`